devicon.json

```diff
@@ -4073,6 +4073,40 @@
         "color": "#00618a",
         "aliases": []
     },
+    {
+        "name": "nano",
+        "altnames": [
+            "gnu-nano",
+            "GNU nano"
+        ],
+        "tags": [
+            "text editor",
```

Comment:
I'm not sure tags should have spaces. 🤔


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
We have a lot of tags with spaces already. Is there any reason we shouldn't have spaces?